src/ampPage.js

```diff
@@ -1,6 +1,11 @@
 import ampScripts from './ampScripts';
 import ampJson from './ampJson';
 
+function ampExperimentToken(token) {
+  if (!token) return '';
+  return `<meta name="amp-experiment-token" content="${token}">\n`;
```

Comment:
This has the same issue (which imo should have blocked the PR) which is that `token` isn't being sanitized, and could be used to accidentally or maliciously inject content into the page.